jupyterlite_xeus_python/env_build_addon.py

```diff
@@ -48,20 +48,32 @@
     MAMBA_PYTHON_AVAILABLE = False
 
 try:
-    check_call(["mamba", "--version"], **SILENT)
-    MAMBA_AVAILABLE = True
+    MAMBA_COMMAND = shutil.which("mamba")
+    if MAMBA_COMMAND:
+        check_call([MAMBA_COMMAND, "--version"], **SILENT)
```

Comment:
Do you think we still need these `check_call` calls? Could `shutil.which` be enough?

Reply:
I think we don't need `check_call`. 

I have not proposed further refactoring just to keep focus on the proposed fix. But if we agree the fix is proper, It makes sense to additionally simplify the code. I have just pushed a commit.

Reply:
Great! Thanks a lot!